packages/in_app_purchase/example/android/app/src/test/java/io/flutter/plugins/inapppurchase/MethodCallHandlerTest.java

```diff
@@ -286,7 +289,6 @@ public void launchBillingFlow_ok_null_AccountId() {
     verify(mockBillingClient).launchBillingFlow(any(), billingFlowParamsCaptor.capture());
     BillingFlowParams params = billingFlowParamsCaptor.getValue();
     assertEquals(params.getSku(), skuId);
-    assertNull(params.getAccountId());
```

Comment:
getAccountId is not available anymore so we can't really test this value, however, we can still test if the call crashes, so ill leave the test here

Reply:
Can you rename the test then, to better explain what it's doing now? (Probably needs a comment in addition to a new name.)

Reply:
done